backend-tests/tests/test_useradm_enterprise.py

```diff
@@ -49,8 +49,8 @@ def clean_migrated_mongo(clean_mongo):
     yield clean_mongo
 
 
-@pytest.fixture(scope="function")
-def tenants_users(clean_migrated_mongo):
+@pytest.fixture(scope="function", params=["os", "professional", "enterprise"])
+def tenants_users(request, clean_migrated_mongo):
```

Comment:
idle comment/I am wondering shall we add a scenario with param `"some-other-plan"`? shall it fail or pass?

Reply:
I don't think so, it will fail as you cannot create a tenant with a non existing plan.